plugin/storage/integration/grpc_test.go

```diff
@@ -132,8 +75,10 @@ func (s *GRPCStorageIntegrationTestSuite) initialize(t *testing.T) {
 }
 
 func (s *GRPCStorageIntegrationTestSuite) cleanUp(t *testing.T) {
-	err := s.factory.Close()
-	require.NoError(t, err)
+	require.NoError(t, s.factory.Close())
+	if s.server != nil {
+		require.NoError(t, s.server.Close())
```

Comment:
Would it make sense to release the server here (s.server = nil)?

Reply:
Hmm.. actually the server (GRPCServer) here is a declaration if the current test suite uses remote storage or a plugin. If it uses remote storage then it closes and starts the server again.

But the server inside GPRCServer struct is the gRPC server that handles requests, and I'll add to release the server there.